service-catalog/generic-service.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
```

Comment:
all those exists in: 
https://github.com/techvelocity/velocity-blueprints/tree/main/references/kubernetes/database-containers
please put it there with the changes required 🙏

Reply:
there are files with the same name. can I replace the existing ones?

Reply:
look at the diff in each.
make sure all are additions